src/pages/settings/Subscription/CardSection/BillingBanner/PreTrialBillingBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Report from '@libs/actions/Report';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ReportUtils from '@libs/ReportUtils';
+import ROUTES from '@src/ROUTES';
+import BillingBanner from './BillingBanner';
+
+function PreTrialBillingBanner() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const navigateToChat = () => {
+        const reportUsedForOnboarding = ReportUtils.getChatUsedForOnboarding();
+
+        if (!reportUsedForOnboarding) {
+            return;
+        }
+
+        Report.openReport(reportUsedForOnboarding.reportID);
```

Comment:
Can we avoid calling `openReport` here as it will get called anyway as part of navigation? Otherwise, it will get called twice unnecessarily.

Reply:
Adjusted!